packages/website/package.json

```diff
@@ -12,24 +12,26 @@
     "@codesandbox/sandpack-react": "^1.17.0",
     "@contentful/f36-components": "^4.41.1",
     "@contentful/f36-docs-utils": "^4.0.2",
+    "@contentful/f36-empty-state": "^4.41.1",
```

Comment:
it can be imported from `f36-components`, right?

Reply:
Yeah true, this change was `npm` automatically reordering things.